### PR TITLE
controls tutorial

### PR DESCRIPTION
**Pull request by norgeous**

### CODE REVIEW

File: src/overlays/controlsTutorial.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+        horiz.destroy();
+        keys.destroy();
+        space.destroy();
+      }, 2_000);
+    } else {
+      balance.destroy();
+      jump.destroy();
+    }
+  }, 2_000);
```
Comment (ScottyXIII) [inline src/overlays/controlsTutorial.ts:79]: Maybe we should keep the keys on screen until they finish the tutorial level?

**Comment by ScottyXIII** [comment]
Wondering if we should make it more obvious that they have to touch the left side of the screen for movement and right for jump? Maybe a dotted line going down the middle of the screen might help with that? 🤔

File: src/overlays/controlsTutorial.ts (hunk 1 above)
Comment (norgeous) [reply src/overlays/controlsTutorial.ts:79]: yeah we could do that... did you want me to make a separate level too?

Comment (ScottyXIII) [reply src/overlays/controlsTutorial.ts:79]: oooh no I thought this was gonna be it's own level but yeah this is better!

**Pull request merged**